client/src/App.spec.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from "@testing-library/react";
+import Cards from './components/Cards';
+
+describe('Card', () => {
+    const exampleData = [
+    {
+        author: 'Max Muster',
+        text: 'First example-text',
+        id: 1,
+    },
+
+    {
+        author: 'Tina Müller',
+        text: 'second example-text',
+        id: 2,
+    },
+    {
+        author: 'Peter Schröder',
+        text: 'third example-text',
+        id: 3,
+    },
+    {
+        author: 'Sabrina Wilde',
+        text: 'fourth example-text',
+        id: 4,
+    },
+    ]
+
+    it('Renders Card', () => {
+        render(
+            <main>
+            {exampleData.map(data => (
```

Comment:
More of a question: is it necessary to test for ALL cards to render?

Reply:
To test this was a recommendation of Stefan. He argued that this test is closer to the actual display of the cards because more than one can be displayed in the lean coffee board.

Reply:
Ok! We changed our test according to your remarks!